usr/src/uts/common/fs/zfs/dsl_scan.c

```diff
@@ -1446,6 +1446,10 @@ dsl_scan_active(dsl_scan_t *scn)
 	return (used != 0);
 }
 
+/*
+ * Called whenever a txg syncs.  Called for both current txgs and old ones that
+ * are being resilvered
```

Comment:
By "current txgs" I mean new transaction groups, caused by new data being written to the pool.  But "old ones" I mean transaction groups that are in the missing range of a damaged vdev and are currently being resilvered.